safe_transaction_service/history/views.py

```diff
@@ -314,15 +317,63 @@ class SafeModuleTransactionListView(ListAPIView):
     pagination_class = pagination.DefaultPagination
     serializer_class = serializers.SafeModuleTransactionResponseSerializer
 
-    def get_queryset(self):
-        return (
-            ModuleTransaction.objects.filter(safe=self.kwargs["address"])
-            .select_related("internal_tx__ethereum_tx__block")
-            .order_by("-created")
-        )
+    _schema_unique_hash = openapi.Parameter(
```

Comment:
@Uxio0 I get your point, I did it with query param because this id is generated from other fields of database. 
I think that your suggestion make sense.